### PR TITLE
More small improvements

### PR DESCRIPTION
**Pull request by spinjo**
### Goals
- More small improvements to make tagging work

### Changes
- Added option `model.add_fourmomenta_backbone` for LLoCa taggers
- Option for global rest frame boosts, `data.boost_jet`
- Add `CosineAnnelingWarmRestarts` scheduler
- Add config files for the small top tagging dataset as used in the paper

### CODE REVIEW

File: lloca/nn/attention.py [hunk 1]
```diff
@@ -142,9 +142,10 @@ def forward(self, q_local, k_local, v_local, **attn_kwargs):
 
         # transform out back into local frame
         out_local = self.transform(out_global, self.lframes)
-        out_local = torch.nn.functional.layer_norm(
-            out_local, normalized_shape=out_local.shape[-1:]
-        )
+        # layer norm can help with stability, but sometimes hurts performance
```
Comment (luigifvr) [inline lloca/nn/attention.py:145]: Does it improve stability? It can also be deleted entirely

File: config/toptagging.yaml [hunk 2]
```diff
@@ -4,6 +4,27 @@ exp_name: topt_local_debug
 data:
  data_dir : data
  dataset: full
+ add_tagging_features_lframesnet: true
```
Comment (spinjo) [inline config/toptagging.yaml:7]: This information is all copied from the `config/tagging.yaml` config if you specify `tagging` in the defaults. I would revert the changes in this file again if you agree, and only keep the `training: tag_top_default`, `model: tag_top_transformer` in the defaults

Comment (spinjo) [reply config/toptagging.yaml]: done
